### PR TITLE
Add parameter description

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/failoverclusters/Get-ClusterVMMonitoredItem.md [hunk 1]
```diff
@@ -138,7 +140,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/failoverclusters/Get-ClusterVMMonitoredItem.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment (velkovb) [reply docset/winserver2012r2-ps/failoverclusters/Get-ClusterVMMonitoredItem.md]: The actual article title is about_CommonParameters with lower case, so I believe we should leave it as is.

Comment (JohanFreelancer9) [reply docset/winserver2012r2-ps/failoverclusters/Get-ClusterVMMonitoredItem.md]: Not the article that this link goes to. It is definitely upper case. Would you mind checking again?

Comment (velkovb) [reply docset/winserver2012r2-ps/failoverclusters/Get-ClusterVMMonitoredItem.md]: image
All about articles start with lower case about.

**Comment by JohanFreelancer9** [comment]
This is where the link takes us, but not a problem. You can leave as is.
[image: About.jpg]

**Pull request merged**